tests_v2/test_cli_new_workspace_external_git_branch_refresh.py

```
@@ -0,0 +1,204 @@
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).parent))
+from cmux import cmux, cmuxError
+
+
+SOCKET_PATH = os.environ.get("CMUX_SOCKET", "/tmp/cmux-debug.sock")
```

Ruff (0.15.4)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-debug.sock"

(S108)